service/src/test/java/bio/terra/drshub/services/DrsProviderServiceTest.java

```diff
@@ -28,4 +28,16 @@ void testResolvesdnsHostsAndProviders() {
       assertEquals(cidProviderHost.drsProvider(), resolvedProvider);
     }
   }
+
+  @Test
+  void testMapsCompactIdsToTheirFullHosts() {
+    for (var entry : config.getCompactIdHosts().entrySet()) {
+      var compactId = entry.getKey();
+      var hostName = entry.getValue();
+      var testCompactUri = String.format("drs://%s/12345", compactId);
+      var expectedUri = String.format("//%s/12345", hostName);
```

Comment:
weird seeing a URI without a scheme/protocol in front! I looked through the code a bit but didn't figure out why it doesn't need the scheme in the URI 🤷‍♂️

Reply:
Neither did I, but its what's already there!